### PR TITLE
Implement nonuk.yml as variables

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

https://trello.com/c/2WHavuTv

### Context

We're centralising all of our monetary values into one place on the service. This PR does this for the values related to candidates not based in the UK.

### Changes proposed in this pull request

- Update any page with a monetary value with the relevant variable.

### Guidance to review

Pages with changes:

[ ] https://get-into-teaching-app-review-4027.test.teacherservices.cloud/non-uk-teachers/get-an-international-relocation-payment
[ ] https://get-into-teaching-app-review-4027.test.teacherservices.cloud/non-uk-teachers/teach-in-england-if-you-trained-overseas
[ ] https://get-into-teaching-app-review-4027.test.teacherservices.cloud/non-uk-teachers/ukraine
[ ] https://get-into-teaching-app-review-4027.test.teacherservices.cloud/non-uk-teachers/visas-for-non-uk-teachers

### Pre-election period restrictions

* [ ] This change is permitted within the constraints of the [pre-election period guidelines](https://www.gov.uk/government/publications/election-guidance-for-civil-servants/general-election-guidance-2024-guidance-for-civil-servants-html#publishing-content-online-)

### CODE REVIEW

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ related_content:
 promo_content:
 navigation: 20.50
 navigation_title: Get an international relocation payment
-navigation_description: Teachers of languages or physics could be eligible for financial help from the UK government worth £10,000.
+navigation_description: Teachers of languages or physics could be eligible for financial help from the UK government worth $nonuk_internationalrelocationpayment_value$.
```
Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:14]: This should be showing on the category page, but is broken https://get-into-teaching-app-review-4027.test.teacherservices.cloud/non-uk-teachers. The value looks correct though?

File: app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md [hunk 2]
```diff
@@ -151,10 +151,10 @@ To be eligible for a high potential individual visa, you must:
 * have a bachelor’s degree, master’s degree or PhD from a university listed in the [Global Universities](https://www.gov.uk/high-potential-individual-visa/eligibility) list
 * have been awarded your qualification no more than 5 years before you apply
 * have English language skills to [B1 intermediate level](https://www.efset.org/cefr/b1/)
-* have personal funds of £1,270
+* have personal funds of $nonuk_highpotentialvisaminimumsavings$
```
Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md:154]: Suggest we change minimum to min here to make it a bit shorter

File: app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md [hunk 3]
```diff
@@ -167,28 +167,28 @@ To be eligible for the India young professionals scheme visa, you must:
 * be an Indian citizen
 * be between 18 and 30 years old
 * have an eligible qualification
-* have £2,530 in savings
+* have $nonuk_indiayoungprofessionalsschememinimumsavings$ in savings
```
Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md:170]: As above, suggest we change minimum to min

File: app/views/content/non-uk-teachers/get-an-international-relocation-payment.md (hunk 1 above)
Comment (gemmadallmandfe) [reply app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:14]: Nav descriptions need to use the other format as they appear on category pages which have a different file format (it's very confusing!)

There is an example on the internships PR https://github.com/DFE-Digital/get-into-teaching-app/pull/4028/files

image

Comment (Sarah-DfE) [reply app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:14]: Note to me change format of variable

Comment (Sarah-DfE) [inline app/views/content/non-uk-teachers/get-an-international-relocation-payment.md:14]:
```suggestion
navigation_description: Teachers of languages or physics could be eligible for financial help from the UK government worth <%= v :nonuk_internationalrelocationpayment_value %>.
```

**Comment by gemmadallmandfe** [comment]
Closing this PR - this will be replaced by two separate PRs

**Pull request closed without merging**